cloud-regionsrv-client.spec

```diff
@@ -210,6 +210,15 @@ gzip %{buildroot}/%{_mandir}/man1/*
 %pre addon-azure
 %service_add_pre regionsrv-enabler.timer
 
+%preun
+# Do not run during an upgrade process
```

Comment:
The comment information that this script code does not run on a package upgrade process should be imho added directly in front of the if condition. It would be even better to give the $1 more meaning by storing the value in a more self explaining variable. In that case the comment can go away completely because reading the code will then be enough. But I leave this part to you.